src/client/components/common/Toggle/index.tsx

```diff
@@ -8,6 +8,7 @@ export interface ToggleButtonProps {
   disabled?: boolean;
   color?: string;
   onClick?: () => void;
+  ariaLabel?: string;
```

Comment:
Curious, any reason for adding this prop instead of just using the native `aria-label`?

Reply:
Yeah probably.